scripts/versions.py

```diff
@@ -525,8 +571,7 @@ def parse_paths(paths: str, version: str) -> list[FilePatternGroup]:
 
         # Verify that the path points to a valid file; skip it if not
         if not os.path.isfile(file_path):
-            print(f"File '{file_path}' does not exist or is not a file. Skipping...")
-            continue
+            error_exit(title="File not found", message=f"File '{file_path}' does not exist or is not a file.")
```

Comment:
we used to continue iterating if the current file path was not valid, but in this updated code we exit, is this correct?